### PR TITLE
Add github action

### PR DESCRIPTION
**Pull request by ryanzhang-oss**

### CODE REVIEW

File: go.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: "0 10 * * 1-5"
```
Comment (wonderflow) [inline go.yml:9]: I think we don't need cron for CI. We need add cron in another action file to build and push image.

**Review by wonderflow — APPROVED** [review]
LGTM

File: go.yml (hunk 1 above)
Comment (ryanzhang-oss) [reply go.yml:9]: The goal is to run multiple times a day just to make sure there are no false positives. I need to tune the cron schedule though.

Comment (wonderflow) [reply go.yml:9]: 👍 OK

**Comment by ryanzhang-oss** [comment]
replace by https://github.com/crossplane/addon-oam-kubernetes-local/pull/12

**Comment by wonderflow** [comment]
this PR could be closed

**Pull request closed without merging**